### PR TITLE
Feat/main/basic server

### PR DESCRIPTION
**Pull request by almahirpm**
Add the basic server config 
-method get
-method post 
-method get with specific pokemon type (Example)

### CODE REVIEW

File: pokekotlin/local.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Wed Jan 06 12:33:25 COT 2021
+sdk.dir=/Users/apinango/Library/Android/sdk
```
Comment (vxcamiloxv) [inline pokekotlin/local.properties:8]: I think we can ignore this local properties

File: pokekotlin/src/Application.kt [hunk 2]
```diff
@@ -1,13 +1,30 @@
 package org.pokekotlin
 
+import com.typesafe.config.ConfigFactory
 import io.ktor.application.*
+import io.ktor.config.HoconApplicationConfig
+import io.ktor.features.*
+import io.ktor.http.*
 import io.ktor.response.*
 import io.ktor.request.*
+import io.ktor.routing.*
```
Comment (vxcamiloxv) [inline pokekotlin/src/Application.kt:10]: what will be our import convention, allowing wildcard or implicit? https://github.com/android/kotlin-guides/issues/65 https://jiga.dev/avoiding-wildcard-imports-in-java-kotlin-with-intellij/ cc: @Tremerex

File: pokekotlin/src/RootRoute.kt [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    fun Routing.rootGet() {
+
+        get("/") {
+            call.respondText("Pokemon with Kotlin + RUST", contentType = ContentType.Text.Plain)
+        }
+
+    }
+
+
+    fun Routing.rootPost(){
```
Comment (vxcamiloxv) [inline pokekotlin/src/RootRoute.kt:21]:
```suggestion
    fun Routing.rootPost() {
```

File: pokekotlin/src/RootRoute.kt [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        post("/") {
+            val post = call.receive<String>()
+            call.respondText("Post request $post", ContentType.Text.Plain)
+        }
+    }
+
+
+    fun Routing.rootGetWaterPokemon(){
```
Comment (vxcamiloxv) [inline pokekotlin/src/RootRoute.kt:29]:
```suggestion
    fun Routing.rootGetWaterPokemon() {
```

File: pokekotlin/src/RootRoute.kt [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        get("/waterPokemon"){
```
Comment (vxcamiloxv) [inline pokekotlin/src/RootRoute.kt:30]:
```suggestion
        get("/waterPokemon") {
```

File: pokekotlin/src/RootRoute.kt [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+            call.respondText(water_pokemon)
+        }
+    }
```
Comment (Tremerex) [inline pokekotlin/src/RootRoute.kt:33]: Add newline please!

**Pull request merged**